src/Pagination.js

```diff
@@ -1,51 +1,44 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import cx from 'classnames';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
 import Icon from './Icon';
 import PaginationButton from './PaginationButton';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
```

Comment:
we like to keep our imports organized, grouped by external and internal dependencies